torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1330,7 +1333,8 @@ class IrParser {
                 : nullptr;
 
             Val* out = clamp(operand, min, max);
-            value_map.emplace(node->output()->unique(), out);
+            value_map.emplace(
```

Comment:
(not asking you to fix anything here, I'm just curious) - is it accurate that this is only necessary as part of the "real fix?" coming from csarofeen#1675? Or is this necessary for the changes below?

Reply:
The few places with the memory marked on outputs are the real fixes for format propagation. (where we generate output with wrong shapes).

Changes below for profiling information are coming from https://github.com/csarofeen/pytorch/pull/1677

Together they should fix the two issues coming from torchvision